samples/subsys/usb/hid-keyboard/src/main.c

```diff
@@ -279,6 +279,17 @@ int main(void)
 			continue;
 		}
 
+		if (usbd_is_suspended(sample_usbd)) {
+			/* on a press of any button, send wakeup request */
+			if (kb_evt.value) {
+				ret = usbd_wakeup_request(sample_usbd);
+				if (ret) {
+					LOG_ERR("Remote wakeup error, %d", ret);
+				}
+			}
+			continue; /* cannot communicate in suspend */
```

Comment:
Please do not add unnecessary comments, and please not to the right of a statement.
```
			}

			continue;
```

Reply:
Done.